packages/hdf5/hdf5.0.1.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Vladimir Brankov <vbrankov@janestreet.com>"
+authors: "Vladimir Brankov <vbrankov@janestreet.com>"
+homepage: "https://github.com/vbrankov/hdf5-ocaml"
+bug-reports: "https://github.com/vbrankov/hdf5-ocaml/issues"
+license: "MIT"
+dev-repo: "git@github.com:vbrankov/hdf5-ocaml.git"
```

Comment:
This needs to be a URI with a scheme. I think you can use this format with `git://` or `git+ssh://` but it is most compatible to put something like `https://github.com/vbrankov/hdf5-ocaml.git`.